### PR TITLE
New Copy for Enrollment Start, Enrollment Success, Help, 404, 500

### LINKED ISSUE #676: Engineering: Load Spanish Translations for new copy
- https://login.gov/es/help/verify-your-identity/how-to-verify-your-identity/
- https://login.gov/es/help/verify-your-identity/overview/
- https://design.login.gov/content/#common-terms-and-phrases

Login.gov Design System translation guide:

English | Spanish | French
-- | -- | --
Authentication app | Aplicación de autenticación | Application d’authentification
Authentication method | Método de autentificación | Méthode d’authentification
Backup codes | Códigos de respaldo | Codes de sauvegarde
Government applications | Aplicaciones gubernamentales | Applications gouvernementales
Government employee ID (PIV/CAC card) | Identificación de empleado del gobierno (tarjeta PIV/CAC) | Numéro d’employé du gouvernement (Carte PIV/CAC)
Personal key | Llave personal | Clé personnelle
Phone | Teléfono | Téléphone
Security key | Llave de seguridad | Clé de sécurité
Two-factor authentication | Autenticación de dos factores | Authentification à deux facteurs

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #676

This PR updates several pages of copy for both English and Spanish:
- Spanish: Elig Start
- Spanish: Enrollment Success
- Spanish: Help
- Eng & Sp: 404
- Eng & Sp: 500
- Eng & Sp: Elig Error

### CODE REVIEW

**Comment by thekaveman** [comment]
Looking at this 👀

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -359,55 +359,55 @@ msgstr "TODO: Computer monitor with checkmark"
...
 msgid "eligibility.pages.start.oauth.required_items[1]"
-msgstr "TODO: Your Social Security number"
+msgstr "Su número de Seguro Social"
 
 msgid "eligibility.pages.start.oauth.required_items[2]"
-msgstr "TODO: A phone number with a phone plan associated with your name"
+msgstr "Un número de teléfono con un plan de teléfono asociado con su nombre"
 
 msgid "eligibility.pages.start.oauth.content_title"
 msgstr "TODO: You will need a few items to connect your discount:"
```
Comment (thekaveman) [inline benefits/locale/es/LC_MESSAGES/django.po:394]: This one is still a `TODO:`, we have a translation in the doc

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -23,7 +23,7 @@ msgstr ""
 
 msgid "core.pages.help.about.p[1]"
 msgstr ""
-"Nos asociamos con Login.gov para confirmar la edad para los descuentos basados en la edad y las agencias de transporte locales para confirmar los programas de descuento específicos. Una vez verificados, los descuentos se adjuntan a su tarjeta bancaria a través de nuestro socio de pago, Littlepay, para que cuando pague su viaje en transporte, obtenga su descuento automáticamente."
+"Nos asociamos con Login.gov para confirmar la edad para los descuentos basados en la edad y  nos asociamos con las agencias de transporte locales para confirmar los programas de descuento específicos. Una vez verificados, los descuentos se adjuntan a su tarjeta bancaria a través de nuestro socio de pago, Littlepay, para que cuando pague su viaje en transporte, obtenga su descuento automáticamente."
```
Comment (thekaveman) [inline benefits/locale/es/LC_MESSAGES/django.po:26]: Super minor but there is an extra space here before `nos`

**Review by thekaveman — CHANGES_REQUESTED** [review]
Made a few minor comments inline.

I think the Enrollment index page is missing some updates (both English and Spanish), [link to section is GDoc](https://docs.google.com/document/d/1Ndai_9Ut6DLU86w6BpkmU23xzG1QF0a0-BeQ3_FWNoQ/edit#bookmark=id.wf8jwf7c04s4)

English:

```markdown
# Let’s link your card to your discount.

Now that you are logged in we can link your transit discount to your card through our payment partner, LittlePay.

We don’t store your information, and you won’t be charged. When you use this card to pay for transit, you will always receive your discount.

Please use a bank-issued contactless Visa or Mastercard credit or debit card.
```

Spanish:

```markdown
# Vamos a vincular su tarjeta con su descuento.

Ahora que ha iniciado sesión, podemos vincular su descuento de tránsito a su tarjeta a través de nuestro socio de pagos, Littlepay.

No almacenamos su información y no se le cobrará. Cuando utilice esta tarjeta para pagar el tránsito, siempre recibirá su descuento.

Utilice una tarjeta de crédito o débito Visa o Mastercard sin contacto emitida por un banco.
```

**Comment by machikoyasuda** [comment]
## Screenshots - Could not be verified page

image
image

**Comment by machikoyasuda** [comment]
@thekaveman Ready for re-review

**Review by thekaveman — APPROVED** [review]
Nice work!! 🎸

**Pull request merged**